### PR TITLE
[Bug] fixed pgsql-schema.sql table field default value bug

### PR DESCRIPTION
**Pull request by youyangkou**
#### Issue Number: [1590](https://github.com/apache/incubator-streampark/issues/1590)
[1591](https://github.com/apache/incubator-streampark/issues/1591)
[1592](https://github.com/apache/incubator-streampark/issues/1592)
#### Problem Summary:
When using postgres as the database, the default value of the database table field is not consistent with the default value of the mysql database table field, resulting in a code logic error
### What is changed and how it works?
Refer to the field default value of mysql-schema.sql to set the field default value of pgsql-schema.sql

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

### CODE REVIEW

File: streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql [hunk 1]
```diff
@@ -101,14 +101,14 @@ create table "public"."t_alert_config" (
...
+  "alert_type" int4 default 0,
   "email_params" varchar(255) collate "pg_catalog"."default",
   "sms_params" text collate "pg_catalog"."default",
   "ding_talk_params" text collate "pg_catalog"."default",
   "we_com_params" varchar(255) collate "pg_catalog"."default",
   "http_callback_params" text collate "pg_catalog"."default",
   "lark_params" text collate "pg_catalog"."default",
-  "create_time" timestamp(6) not null,
+  "create_time" timestamp(6) not null default current_timestamp,
   "modify_time" timestamp(6) not null
```
Comment (lvshaokang) [inline streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql]: Could you confirm that the modify_time field whether need default value?

Comment (youyangkou) [reply streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql]: @lvshaokang Thank you for reminding me. `modify_time` can also set the default value, which means that `create_time` and `modify_time` are equal when new records are added, and modify_time must be updated manually for subsequent changes.

Comment (youyangkou) [reply streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql]: Thanks for your review, addressed.

**Comment by youyangkou** [comment]
@wolfboys @lvshaokang PTAL

**Comment by wolfboys** [comment]
we need to unify the definitions of `create_time` and `modify_time`

in mysql ddl: 

`
create_time timestamp(3) not null default current_timestamp(3) comment 'create time',
modify_time timestamp(3) not null default current_timestamp(3) on update current_timestamp(3) comment 'modify time',
`

----

in pgsql ddl:

`
"create_time" timestamp(6) not null default timezone('UTC-8'::text, (now())::timestamp(0) without time zone),
"modify_time" timestamp(6) not null default timezone('UTC-8'::text, (now())::timestamp(0) without time zone)
`

---

in h2 ddl:

`
create_time timestamp not null default current_timestamp comment 'create time',
modify_time timestamp not null default current_timestamp on update current_timestamp comment 'modify time'
`

**Comment by wolfboys** [comment]
You can continue to improve mysql and h2 ddl schema, see https://github.com/apache/incubator-streampark/pull/1598#issuecomment-1246189783

File: streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql [hunk 2]
```diff
@@ -147,13 +148,13 @@ create table if not exists `t_flink_env` (
   `flink_conf` text  not null comment 'flink-conf',
   `is_default` tinyint not null default '0' comment 'whether default version or not',
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql]: `default '0' ` can be changed to `default 0`

File: streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql [hunk 3]
```diff
@@ -375,7 +376,7 @@ create table if not exists `t_flink_cluster` (
   `resolve_order` tinyint default null,
   `exception` text comment 'exception information',
   `cluster_state` tinyint default '0' comment 'cluster status (0: created but not started, 1: started, 2: stopped)',
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql]: default '0' can be changed to default 0

Comment (youyangkou) [reply streampark-console/streampark-console-service/src/main/resources/db/schema-h2.sql]: > default '0' can be changed to default 0

Thanks for your review, addressed.

File: streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql [hunk 4]
```diff
@@ -281,22 +293,22 @@ create table "public"."t_flink_cluster" (
...
   "k8s_namespace" varchar(255) collate "pg_catalog"."default",
   "service_account" varchar(50) collate "pg_catalog"."default",
   "description" varchar(255) collate "pg_catalog"."default",
   "user_id" int8,
   "flink_image" varchar(255) collate "pg_catalog"."default",
   "dynamic_options" text collate "pg_catalog"."default",
-  "k8s_rest_exposed_type" int2,
-  "k8s_hadoop_integration" boolean,
-  "flame_graph" boolean,
+  "k8s_rest_exposed_type" int2 default 2,
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql:304]: Do we need to set a default value?

Comment (youyangkou) [reply streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql:304]: > Do we need to set a default value?

Yes, the k8s_rest_exposed_type default value is  nodeport, which is also consistent with mysql-schema.sql

Comment (wolfboys) [reply streampark-console/streampark-console-service/src/assembly/script/schema/pgsql-schema.sql:304]: ok

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**